test/decl/ext/extensions.swift

```diff
@@ -37,6 +37,19 @@ protocol NestingTest5 {
 func nestingTest6() {
   extension Foo {} // expected-error {{declaration is only valid at file scope}}
 }
+extension Array {
```

Comment:
Might be better to put the original test case in validation-test/compiler_crashers_2_fixed/sr15447.swift.

Reply:
Let's keep this one as well to spot regressions or improvements in diagnostics.